app/controllers/RepoAcceptListService.scala

```diff
@@ -28,7 +28,7 @@ class RepoAcceptListService()(implicit
   } yield treeT.map(_.tree.exists(_.path.endsWith(ProutConfigFileName))).getOrElse(false)
 
   private def getAllKnownRepos: Future[RepoAcceptList] = for { // check this to see if it always expends quota...
-    allRepos <- github.listReposAccessibleToTheApp.allItems()
+    allRepos <- github.listReposAccessibleToTheApp().allItems()
```

Comment:
Scala 3 [dropped 'auto-application'](https://docs.scala-lang.org/scala3/reference/dropped-features/auto-apply.html) of empty parentheses (eg letting you omit the empty `()` when calling a method that has an empty parameter list in it's method definition) - so the application syntax (the way you _call_ a method) has to follow exactly the parameter syntax (the way the method is defined). Otherwise we get a `[E100] Syntax Error`:

```
[error] -- [E100] Syntax Error: /Users/roberto_tyley/code/prout/app/controllers/RepoAcceptListService.scala:31:23 
[error] 31 |    allRepos <- github.listReposAccessibleToTheApp.allItems()
[error]    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[error]    |method listReposAccessibleToTheApp in class GitHub must be called with () argument
```